File: aula05_exercicio.py

preço = 150
promoçao = 20
preço_promoçao = preço - promoçao
print (preço_promoçao)

peso = 70
altura = 1.75
imc = peso /( altura **2 )
print ( imc )

idade = 22
tem_experiencia = True 
elegivel_concurso = idade  >= 18 and tem_experiencia
print(" elegivel ao concurso?", elegivel_concurso)

nivel_acesso = int(input("seu nivel de acesso") )
if nivel_acesso == 1:
    print("acesso restrito") 
elif nivel_acesso == 2:
    print("acesso parcial")
elif nivel_acesso == 3:
    print("acesso total")

celsius = int(input(" digite sua temperatura"))
fahrenheit = (celsius * 9/5) + 32
print(fahrenheit)

um = int(input("Digite o número para saber se é par ou ímpar: "))

if num % 2 == 0:
    print(f"O número {num} é Par!")
else:
    print(f"O número {num} é Ímpar!")

peso = int(input("Digite o peso do Frete: "))

if peso <= 5:
    frete = 20.00
elif peso > 10:
    frete = 30.00

print(f'O valor do Frete é {frete}')

File: aula05_exercicio.js

let preco = 150;
let desconto = 20;
let valor_final = preco - desconto;
console.log("preco - desconto =", valor_final);


let peso = 70;
let altura = 1.75;
let imc = peso / (altura ** 2);
console.log("IMC:", imc);


let idade = 22;
let tem_experiencia = true;
let elegivel_concurso = idade >= 18 && tem_experiencia;
console.log("Elegível ao concurso?", elegivel_concurso);



let nivel_acesso = parseInt(prompt("Digite seu nível de acesso:"));
if (nivel_acesso === 1) {
    console.log("Acesso restrito");
} else if (nivel_acesso === 2) {
    console.log("Acesso Parcial");
} else if (nivel_acesso === 3) {
    console.log("Acesso Total");
} else {
    console.log("Nível de acesso inválido");
}


let celsius = parseInt(prompt("Digite sua temperatura em Celsius:"));
let fahrenheit = (celsius * 9/5) + 32;
console.log("Temperatura em Fahrenheit:", fahrenheit);

let valor_inteiro = parseInt(prompt("Digite um número:"));